spec/helpers/job_application_helper_spec.rb

```diff
@@ -152,4 +152,27 @@
       end
     end
   end
+
+  describe "#job_application_view_applicant" do
+    subject { helper.job_application_view_applicant(vacancy, job_application) }
+
+    let(:vacancy) { create(:vacancy, :published) }
+
+    context "when job application is withdrawn" do
+      let(:job_application) { create(:job_application, :status_withdrawn) }
+
+      it "returns the applicant name" do
+        expect(subject).to eq(tag.span("#{job_application.first_name} #{job_application.last_name}", class: "govuk-!-font-size-19"))
```

Comment:
I think we can dial down the specificity here - maybe just check that the text is there?